From b627220ef6896ef520dbe917893e3071b9ccd444 Mon Sep 17 00:00:00 2001
From: Sudeep Holla <Sudeep.Holla@arm.com>
Date: Mon, 14 Sep 2015 16:01:54 +0100
Subject: [PATCH 129/133] ACPI: int340x_thermal: add missing CONFIG_ prefix

commit cf3a51059efd653aa38667db799e85d77231af9d upstream.

This patch adds the missing CONFIG_ prefix to INTEL_SOC_DTS_THERMAL
macros.

Signed-off-by: Sudeep Holla <sudeep.holla@arm.com>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
Signed-off-by: Pengyu Ma <pengyu.ma@windriver.com>
---
 drivers/acpi/int340x_thermal.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/acpi/int340x_thermal.c b/drivers/acpi/int340x_thermal.c
index 9dcf836..6c7c797 100644
--- a/drivers/acpi/int340x_thermal.c
+++ b/drivers/acpi/int340x_thermal.c
@@ -35,7 +35,7 @@ static int int340x_thermal_handler_attach(struct acpi_device *adev,
 {
 #if defined(CONFIG_INT340X_THERMAL) || defined(CONFIG_INT340X_THERMAL_MODULE)
 	acpi_create_platform_device(adev);
-#elif defined(INTEL_SOC_DTS_THERMAL) || defined(INTEL_SOC_DTS_THERMAL_MODULE)
+#elif defined(CONFIG_INTEL_SOC_DTS_THERMAL) || defined(CONFIG_INTEL_SOC_DTS_THERMAL_MODULE)
 	/* Intel SoC DTS thermal driver needs INT3401 to set IRQ descriptor */
 	if (id->driver_data == INT3401_DEVICE)
 		acpi_create_platform_device(adev);
-- 
2.0.2

